packages/components/src/toolbar-button/README.md

```diff
@@ -0,0 +1,26 @@
+# ToolbarButton
+
+A ToolbarButton can be used to add actions to your block control, usually inside a ToolbarGroup. It has the same features of the [Button](https://developer.wordpress.org/block-editor/components/button/) component, only its style is more appropriate for a Toolbar.
```

Comment:
> only its style is more appropriate for a Toolbar.

Hey @mateuswetah, thanks for working on this! 

I think there are a few extra features of the `ToolbarButton` over just styling. It also has some additional accessibility functionality implemented via the Reakit library - https://reakit.io/docs/toolbar/. As mentioned there, it implements this toolbar pattern for keyboard interactions - https://www.w3.org/TR/wai-aria-practices/#toolbar.

```suggestion
A ToolbarButton can be used to add actions to your block control, usually inside a ToolbarGroup. It has similar features to the [Button](https://developer.wordpress.org/block-editor/components/button/) component. Using `ToolbarButton` will ensure the correct styling for a button in a toolbar, and also that keyboard interactions in a toolabr are consistent with the [WAI ARIA toolbar pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar).
```

Reply:
Awesome, thanks for the clarification, I think its better :)